<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/ico.png</normaloff>:/ico.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabFirewall">
       <attribute name="title">
        <string>Êñ≠ÁΩë</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>‰ª£Ë®Ä‰∫∫ÔºöAdstargod
ÊòüÁ•ûÊñ≠ÁΩëÔºåÂ•ΩÁî®Ê≤°ÁÉ¶ÊÅº
üòã</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbSound">
          <property name="text">
           <string>Êí≠ÊîæÊèêÁ§∫Èü≥</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartFirewall">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHeadShot">
       <attribute name="title">
        <string>ÁàÜÂ§¥Êï∞</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labHeadShotCount">
          <property name="font">
           <font>
            <pointsize>26</pointsize>
           </font>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartHeadShot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÁÇπÂáªÂêØÂä®</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ËÆ°Êó∂Âô®</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labTimer">
          <property name="text">
           <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;00:00&lt;/span&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;.&lt;/span&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartTimer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÁÇπÂáªÂêØÂä®</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPauseTimer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ÁÇπÂáªÊöÇÂÅú</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Â∏ÆÂä©(&amp;H)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>ËµûÂä©‰ΩúËÄÖ</string>
     </property>
     <addaction name="actionAliPay"/>
     <addaction name="actionAiFaDian"/>
     <addaction name="actionBuyMeACoffee"/>
    </widget>
    <addaction name="actionLogDir"/>
    <addaction name="actionLog"/>
    <addaction name="separator"/>
    <addaction name="actionGitHub"/>
    <addaction name="actionHelpTranslate"/>
    <addaction name="separator"/>
    <addaction name="menu"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Â∑•ÂÖ∑(&amp;T)</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>ËßÜÂõæ(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuDisplayInfo">
     <property name="title">
      <string>‰ø°ÊÅØÂ±ïÁ§∫</string>
     </property>
     <widget class="QMenu" name="menuServer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Â§öÁ´ØÊòæÁ§∫ÔºàÂºÄÂèë‰∏≠Ôºâ</string>
      </property>
      <addaction name="actionEnableServer"/>
      <addaction name="separator"/>
      <addaction name="actionCopyHostAddress"/>
      <addaction name="actionOpenBrowser"/>
     </widget>
     <addaction name="actionDisplayInfo"/>
     <addaction name="actionDisplayInfoTouchable"/>
     <addaction name="separator"/>
     <addaction name="menuServer"/>
    </widget>
    <addaction name="menuDisplayInfo"/>
   </widget>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAboutQt">
   <property name="text">
    <string>ÂÖ≥‰∫é Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>ÂÖ≥‰∫éÊú¨ËΩØ‰ª∂...</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>ËÆøÈóÆ GitHub ‰ªìÂ∫ì...</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>ËÆæÁΩÆ(&amp;S)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Êü•ÁúãÊó•Âøó...</string>
   </property>
  </action>
  <action name="actionLogDir">
   <property name="text">
    <string>ÊâìÂºÄÊó•ÂøóÊñá‰ª∂Â§π...</string>
   </property>
  </action>
  <action name="actionDisplayInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ÊòæÁ§∫...</string>
   </property>
  </action>
  <action name="actionDisplayInfoTouchable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ÂèØÊãñÂä®</string>
   </property>
  </action>
  <action name="actionHelpTranslate">
   <property name="text">
    <string>Â∏ÆÂä©Êàë‰ª¨ËøõË°åÁøªËØë...</string>
   </property>
  </action>
  <action name="actionCopyHostAddress">
   <property name="text">
    <string>Â§çÂà∂Âú∞ÂùÄ</string>
   </property>
  </action>
  <action name="actionEnableServer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ËøêË°å</string>
   </property>
  </action>
  <action name="actionOpenBrowser">
   <property name="text">
    <string>Âú®ÊµèËßàÂô®‰∏≠ÊâìÂºÄ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>ÈÄÄÂá∫(&amp;X)</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/ico.png</normaloff>:/ico.png</iconset>
   </property>
   <property name="text">
    <string>ÊòæÁ§∫(&amp;S)...</string>
   </property>
  </action>
  <action name="actionAiFaDian">
   <property name="text">
    <string>Áà±ÂèëÁîµ...</string>
   </property>
  </action>
  <action name="actionBuyMeACoffee">
   <property name="text">
    <string>Buy Me A Coffee...</string>
   </property>
  </action>
  <action name="actionAliPay">
   <property name="text">
    <string>ÊîØ‰ªòÂÆù...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
